README.md

```diff
@@ -158,15 +165,15 @@ isSomeFeatureEnabled:
     - value: false
 ```
 ```javascript
-var rule = {
+const rule = {
   type: 'list',
   input: 'user.state', //Input can be a nested property
   property: 'states'
 };
 ```
 
 ### Equal
-The equal rule is much like the list rule, except it only allows a single value instead of a list of values.  One common use for this rule would be enabling features based on a single user characteristic that only has a limited number of states (e.g. boolean).  The example below shows you how you would enable a feature for any user that has been using your application for a many years, and is over the age of 18.  This example also debuts another feature of rules: the ability to accept a function in the input property.  Note:  All three rules let you do pass a function for input.
+The equal rule is much like the list rule, except it only allows a single value instead of a list of values.  One common use for this rule would be enabling features based on a single user characteristic that only has a limited number of states (e.g. boolean).  The example below shows you how you would enable a feature for any user that has been using your application for a number years, and is over the age of 18.  This example also shows another feature of rules: the ability to accept a function in the input property.  Note: All three rules let you pass a function for input.
```

Comment:
`Note: All three rules let you pass a function for input.`
This note might best be placed at the top of the section since it applies to all rules (L138)